src/api/index.js

```diff
@@ -6,9 +6,15 @@ export const refresh = () => logined.post('user/refresh');
 
 export const checkNickname = (nickName) => logined.get(`/user/nickname-check?nickname=${nickName}`);
 
-export const checkAccount = (account) => logined.get(`user/account-check?account=${account}-`);
+export const checkAccount = (account) => logined.get(`user/account-check?account=${account}`);
 
 export const checkEmail = (email) => logined.get(`user/email-check?email=${email}`);
 
+export const changeNickName = (nickName) => logined.post('user/nickname', nickName);

```

Comment:
6번째 줄은 `Nickname` 인데 해당 line 은 NickName 이네요 
어떤걸 사용해도 무방아지만 통일성있는 네이밍을 작성하시는게 좋아보입니다.

```suggestion
export const changeNickname = (nickname) => logined.post('user/nickname', nickname);
```

Reply:
082ffb8